hbase_storage/src/main/java/com/splicemachine/storage/HServerLoad.java

```diff
@@ -60,7 +60,7 @@ public long totalRequests(){
         for(Map.Entry<byte[],RegionLoad> regionLoad:regionsLoad.entrySet()){
             String name = Bytes.toString(regionLoad.getKey());
             RegionLoad rl = regionLoad.getValue();
-            PartitionLoad pl = new HPartitionLoad(name,rl.getStorefileSizeMB(),rl.getMemStoreSizeMB(),rl.getStorefileIndexSizeMB());
+            PartitionLoad pl = new HPartitionLoad(name,rl.getStorefileSizeMB()*1024,rl.getMemStoreSizeMB()*1024);
```

Comment:
Should we multiple 1024*1024 to get the number in bytes?

Reply:
Good catch @yxia92 , addressed in a new commit